.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
You've probably committed the content of .idea before adding this line... because the content of .idea is committed. You definitely want to remove it.